lib/elliptic/utils.js

```diff
@@ -147,4 +144,66 @@ function getJSF(k1, k2) {
 
   return jsf;
 }
+
+function lazyComputed(obj, name, computer) {
+  var key = '_' + name;
+  obj.prototype[name] = function lazyComputed() {
+    return this[key] !== undefined ? this[key] :
+           this[key] = computer.apply(this, arguments);
+  };
+}
+
+function parseBytes(bytes) {
+  return typeof bytes === 'string' ? utils.toArray(bytes, 'hex') :
+                                     bytes;
+}
+
+function intFromLE(bytes) {
```

Comment:
That would be great! I opened up some issues re: toArray and honouring the `le` when passing in array.